src/anyvar/utils/liftover_utils.py

```diff
@@ -1,14 +1,24 @@
 """Defines functions used to lift over variants between GRCh37 & GRCh38"""
 
-import re
+import copy
+from enum import Enum
+from typing import TypeVar
 
-from agct import Converter, Genome, Strand
-from ga4gh.vrs.dataproxy import _DataProxy
+from agct import Converter, Strand
+from bioutils.accessions import chr22XY
```

Comment:
I had to check this, but this is the first time we have `bioutils` as a named runtime dependency so we should add it to `pyproject.toml` (it's already implicitly there since it's required by vrs-python/seqrepo but best practice to include it explicitly)